### PR TITLE
add a robot chassis

### PR DESCRIPTION
**Pull request by tianmachashui**

### CODE REVIEW

File: _posts/2018-07-09-dalurobot.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- 'omnidirectional'
+- 'wheeled'
+- 'motor'
+twitter_text: "Checkout the Dalurobot"
+website: http://www.dalurobot.com/web/index.html
+wiki_homepage: wiki.ros.org/Robots/Dalurobot
+---
+
+# Dalurobot_chassis
+
```
Comment (tfoote) [inline _posts/2018-07-09-dalurobot.md]: Could you add some content here?
The main body of the page here is blank except for the heading.

File: _posts/2018-07-09-dalurobot.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+color: '#60bf56'
+tags:
+- 'mobile base'
+- 'logistics robot'
+- 'omnidirectional'
+- 'wheeled'
+- 'motor'
+twitter_text: "Checkout the Dalurobot"
+website: http://www.dalurobot.com/web/index.html
+wiki_homepage: wiki.ros.org/Robots/Dalurobot
```
Comment (tfoote) [inline _posts/2018-07-09-dalurobot.md]: This page does not appear to exist, and it should have the protocol in front `http://`

**Comment by tianmachashui** [comment]
We will fix these problem as soon as possible and thanks

**Comment by tianmachashui** [comment]
we add some information and please check again. THANKS.

**Pull request merged**